qermit/clifford_noise_characterisation/ccl.py

```diff
@@ -35,7 +35,7 @@
     _PolyCDRCorrect,
     cdr_quality_check_task_gen,
 )
-import numpy as np
+import numpy as np  # type: ignore
```

Comment:
These ignores are to do with an updated version of mypy/numpy?

Reply:
Yeah, new numpy has bad type annotations somewhere so I decided to just ignore them